Monika After Story/game/script-topics.rpy

```diff
@@ -15672,3 +15672,97 @@ label monika_eating_meat:
     extend 3hub "That way I can still enjoy your cooking!"
     m 3eua "Whatever we eat, the most important thing to me is that we try to put a little thought into where our food comes from."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_player_eyesight",
+            category=["you"],
+            prompt="Wearing glasses",
+            random="True"
```

Comment:
This should be a boolean, not a string
```suggestion
            random=True
```

Reply:
\\>Copy the code properly
I didn't copy a thing, I just brought in the suggestions.

Reply:
![Capture](https://user-images.githubusercontent.com/53382877/95702435-fa3cb200-0c54-11eb-94ef-8e563ee98000.PNG)